views/includes/check-your-answers/verification-checks.html

```diff
@@ -25,8 +26,9 @@
     {% set partnerChangeLink = OE_CONFIGS.OVERSEAS_ENTITY_DUE_DILIGENCE_CHANGE_PARTNER_NAME %}
   {% endif %}
 {% else %}
+  {% set whoIsCompletingText = "Who is completing this update?" %}
```

Comment:
Question out to Matthew on if we want the "?"

Reply:
Did he confirm this?